temoa/utilities/graphviz_generator.py

```
@@ -1,16 +1,139 @@
+        '--region',
+        action='store',
+        dest='region',
+        help='The region for which the graph is to be generated',
+        default=None,
+    )
+
+    options = parser.parse_args(args)
+
+    if (options.scenario_name is not None) ^ (options.period is not None):
+        parser.print_help()
+        raise ValueError('Scenario and input year must both be present or not present together')
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)